src/cursorless.talon

```diff
@@ -1,64 +1,64 @@
 app: vscode
 -
 {self.simple_cursorless_action} <user.cursorless_target>:
-    user.cursorless_single_target_command(simple_cursorless_action, cursorless_target)
+	user.cursorless_single_target_command(simple_cursorless_action, cursorless_target)
 
 def show <user.cursorless_target>:
```

Comment:
This is probably out of scope for this PR, but I think we could have a list like:

```py
ctx.lists["self.cursorless_extra_action"] = {
    "def show": "editor.action.revealDefinition",
    "ref show": "references-view.find",
}
```

And then we could replace most of the commands by:

```
<user.cursorless_extra_action> <user.cursorless_target>:
    user.cursorless_set_selection_then_run_command(cursorless_target, cursorless_extra_action)
```

Then implement `cursorless_set_selection_then_run_command` to set selection and run the command.  It could take an optional sleep length to let it also replace the quickfix command, that requires a sleep

Tbh we could even have a cursorless action that takes a command as an argument and do it extension-side.  But the point is once we have this capture set up we have more flexibility to tweak it

Reply:
I totally agree we have a lot of flexibility with this solution.

Reply:
does the name `extra_action` make sense?  couldn't come up with a good name

Reply:
btw the other advantage of having a list for these is we can put them in the cheatsheet more easily

Reply:
actions_misc?

Yes that is true for our purpose.